corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -176,8 +176,7 @@ def process_changes_chunk(self, changes_chunk):
                     self.index_info.alias,
                     self.index_info.type,
                     es_actions,
-                    raise_on_error=False,
-                    raise_on_exception=False,
+                    raise_errors=False,
```

Comment:
Curious what is the difference between `raise_on_error` and `raise_on_exception` is?

Reply:
Basically, `raise_on_error` sets whether you want the `bulk()` helper function to raise `BulkIndexError` exceptions or not (e.g. if the backend bulk request has errors that the bulk helper function anticipates), while `raise_on_exception` sets whether you want it to propagate `TransportError` exceptions (e.g. backend bulk request raises an exception).